src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -59,9 +59,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableConfigurationBindingGenerator Condition="'$(EnableConfigurationBindingGenerator)' == ''">true</EnableConfigurationBindingGenerator>
   </PropertyGroup>
 
-  <!-- Enable the interceptors compiler feature by default for projects that use the RequestDelegateGenerator or the ConfigurationBindingGenerator. -->
-  <PropertyGroup Condition="'$(EnableRequestDelegateGenerator)' == 'true' Or '$(EnableConfigurationBindingGenerator)' == 'true'">
-    <Features>$(Features);InterceptorsPreview</Features>
+  <!-- Set the namespaces emitted by the RequestDelegateGenerator for interception when applicable. -->
+  <PropertyGroup Condition="'$(EnableRequestDelegateGenerator)' == 'true'">
```

Comment:
Did you consider setting the conditions on the properties instead, and using the same group for both?